UCLALibrary/fester
# [IIIF-745] Add base URL configuration option

Changes:
* update `GetManifestHandler` to use `S3BucketVerticle` instead of the S3 client
* update `S3BucketVerticle`
  * store manifests with all IIIF Presentation API resource URLs prefixed with a placeholder base URL                                                      
  * by default, return manifests with all instances of the placeholder base URL replaced with the actual base URL
* move `PostCsvHandler` tests to the FIT framework
* update URLs in the test fixtures
* update tests accordingly
* remove erroneous HTTP 400 entry in OpenAPI spec

Note: declarations of `myUrl` and `myUrlPlaceholderPattern` are duplicated a lot more than I'd like, but I expect this to change as we adopt the new FIT framework.
